obfuscate_and_normalize_test.go

```diff
@@ -210,6 +210,21 @@ multiline comment */
 				WithDBMS(DBMSSQLServerAlias1),
 			},
 		},
+		{
+			// [] quoted table name
```

Comment:
Super minor but since this test is identical to the one above it, could we update the comment to make it clearer that the test is meant to cover the other alias. I could see it accidentally being cleaned up as a duplicate in the future since it's just 1 character different than the previous. Took me a second to realize what this was covering